backend/src/services/services.ts

```diff
@@ -66,31 +69,44 @@ const vendor = new Vendor({ vendorRepository });
 
 const http = new Http();
 
-const udemy = new Udemy({
+const udemyCourse = new UdemyCourse({
+  httpService: http,
+  baseUrl: ENV.UDEMY.BASE_URL,
+});
```

Comment:
```suggestion
const udemy = new UdemyCourse({
  httpService: http,
  baseUrl: ENV.UDEMY.BASE_URL,
});
```
should be one service per one vendor